latest/root/usr/bin/run-configsvr

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+#
+# This serves as a container entrypoint for running a MongoDB sharding configuration server. See the README for required ENV variables
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# shellcheck source=/dev/null
+source ${CONTAINER_SCRIPTS_PATH:-}/environment
+
+# shellcheck source=/dev/null
+source ${CONTAINER_SCRIPTS_PATH:-}/generate_container_user
+
+# shellcheck source=/dev/null
+source ${CONTAINER_SCRIPTS_PATH:-}/db/replset/configsvr/init-configsvr
```

Comment:
subroutines are organized hierarchically. run-configsvr references /db/replset/configsvr/init-configsvr, which in turn references /db/replset/common.sh (which contains common replica set functions), which in turn references /db/common.sh (which contains global run-time functions)